repos/system_upgrade/el7toel8/actors/vimmigrate/actor.py

```diff
@@ -0,0 +1,43 @@
+
+from leapp.actors import Actor
+from leapp.libraries.actor import library
+from leapp.libraries.common.reporting import report_with_remediation
+from leapp.models import Report, InstalledRedHatSignedRPM
+from leapp.tags import ApplicationsPhaseTag, IPUWorkflowTag
+
+
+class VimMigrate(Actor):
+    """
+    Modify Vim configuration files to keep the the same behavior as vim-7.4 has.
+    """
+
+    name = 'vim_migrate'
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report,)
+    tags = (ApplicationsPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        error_list = []
+
+        # FIXME: use has_pkg from repo shared library once it is implemented
```

Comment:
The FIXME is related to the https://github.com/oamg/leapp-repository/pull/201 PR